disbatch/createDisBTasks.py

```diff
@@ -0,0 +1,68 @@
+import glob, json, os, os.path, sys
+
+#You need need to modify INPUT_DIR, TASKOUT_DIR to point to the right input and output directory
+INPUT_DIR   = '/mnt/ceph/users/humanbase/data/meta/datasets/'
+TASKOUT_DIR = '~/ceph/projects/MetaSRA-pipeline/data/'
```

Comment:
Use argeparse to pass these dir names on the command line https://docs.python.org/3/library/argparse.html